### PR TITLE
my location

### PR DESCRIPTION
**Pull request by jaztriumph**

### CODE REVIEW

**Comment by jaztriumph** [comment]
closes #1824

**Comment by jaztriumph** [comment]
closes #1824

**Comment by jaztriumph** [comment]
@githubteacher help

File: _pins/jayanth.jason [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (crichID) [inline _pins/jayanth.jason]: @jaztriumph you are really close! Here is what needs to be fixed:

(1) The file extension should be `.json` not `.jason`
(2) you will need to add a space after each `:`
(3) Your GitHub handle should not have a space!

Please let me know if you have any additional questions!

**Pull request merged**